packages/database/src/core/BaseDatabase.js

```diff
@@ -67,6 +67,10 @@ const supportedFunctions = ['ST_AsGeoJSON', 'COALESCE'];
 
 const RAW_INPUT_PATTERN = /(^CASE)|(^to_timestamp)/;
 
+// no math here, just hand-tuned to be as low as possible while
+// keeping all the tests passing
+const HANDLER_DEBOUNCE_DURATION = 250;
```

Comment:
I know this isn’t your change and also _super_ nitpicky, but for clarity I might suggest:

```suggestion
const HANDLER_DEBOUNCE_MS = 250;
```